apps/aens/src/aens_utils.erl

```diff
@@ -7,15 +7,16 @@
 
 -module(aens_utils).
 
+-include("aens.hrl").
+
 %% API
 -export([check_name_claimed_and_owned/3,
-         to_ascii/1]).
-
-%%%===================================================================
-%%% Types
-%%%===================================================================
-
--define(LABEL_SEPARATOR, <<".">>).
```

Comment:
I expect conflicts in master. Leading to multiple PRs rebased and re-reviewed multiple times... May I suggest to split out of this PR all preparation bits (e.g. refactoring)?